roles/container-engine/docker/molecule/default/molecule.yml

```diff
@@ -0,0 +1,38 @@
+---
+driver:
+  name: delegated
+lint:
+  name: yamllint
+  # Disabled until issues are fixed
```

Comment:
Which issues?

Reply:
Thank you for the comment, @MarkusTeufelberger 
I have included the root yamllint configuration and enabled linting in Molecule again.